### DISCUSSION TITLE
Traitor PDAs should be able to track item objectives, suggested by SpBrian on Discord.

### DISCUSSION THREAD

**Discussion by PetMudstone** [unitystation/unitystation#7897] [Ideas]
## Description
'I have had a thought about the item traitor objectives and being able to use your PDA to locate the item instead of having to break into command, sec etc.
What if you are able to ping the item on your PDA, similar to the nuke pointer, however the longer you leave it on, the easier it becomes for Centacom to triangulate your signal and send a message to the command consoles, leading to an arrest. So if it is left continuously on for a period time before being turned off, the message sent changes.
1 minutes to 2 minutes - Print out says "We have detected unusual signals"
3 minutes to 4 minutes - Print out says "We have determined the signal to be tracking an object/person but are not sure exactly what it is"
5 minutes and over - Print out says "We have found the signal to be emanating from X's PDA"'

**Above quoted from SpBrian on Discord.**

**Comment by VernaSaurus** [comment]
I suggested this first >:V
no hard feelings though lmao
